### PR TITLE
I18N Upgrades

### PR DESCRIPTION
**Pull request by DocBadTouch**
**What does this do?**

1.  Adds the basic framework for I18N locolization.
2.  Adds FAQ section to the home page.
3.  Implements the I18N functionality on the Body and FAQ section of the website. 
4.  Adds the locales folder to "public" with sub-folders "en" and "sp" for English and Spanish respectively.
5.  Adds the body.json, faq.json, header.json, main.json, roadmap.json, and tokenomics.json files to the "public/locales/en" directory for I18N resourcing for each section of the home page. 
6.  Adds the next-i18next.config.js file which is used to define parameters for I18N localization.

**New Libraries and why**

1.  "react-collapse"   Used for FAQ 
2.  "react-tabs"  Used for FAQ
3.  "next-i18next" (the primary library used for i18N)
4.  "react-i18next" (possibly removable, but could be used for client-side rendered content)
5.  "i18next-http-backend" (again might not need, but can be used for lazy loading of translation files as we grow)
6.  "i18next-browser-languagedetector" (currently not used, need to figure out implementation, but auto detects users locale)
7.  i18next (possible overlap and un-needed because of the next-i18next library)

**What is left?**

1.  Implement I18N in the rest of the home page sections following the same concept as FAQ and BODY
2.  Set up dynamic data retrieval for contract info (TVL, Price, Vol, etc...) 
3.  possibly implement a translation extraction tool for long-term management of the translation files.  [Locize](https://locize.com/) is an option that would allow our interpreters to directly contribute to missing or needed translations in a user-friendly UI.

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -187,4 +187,49 @@ body {
...
+.faq-title-text {
+  @apply text-6xl font-bold 
+}
+.react-tabs{
+  width: 100%;
+  text-align: center;
+}
+.faq-tab-title {
+  border-bottom: 1px solid #ccc;
+  @apply text-3xl font-bold
```
Comment (snrou) [inline styles/globals.css:206]: cool how you can apply tailwind and regular css in the same class statement 👀

**Review by snrou — APPROVED** [review]

File: pages/index.tsx [hunk 2]
```diff
@@ -6,11 +6,24 @@ import { Body } from '../components/Body'
...
+    },
+  };
+}
+
 
 const Home: NextPage = () => {
   return (
     <div>
-      <Head>
+      <Head children={''}>
```
Comment (snrou) [inline pages/index.tsx:26]: I think this is causing the build issue, try removing prop

**Review by snrou — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (DocBadTouch) [reply pages/index.tsx:26]: Will do.

Comment (DocBadTouch) [reply pages/index.tsx:26]: For some reason when I remove that the compiler throughs an error? 
image

File: styles/globals.css (hunk 1 above)
Comment (DocBadTouch) [reply styles/globals.css:206]: Yeah, I love it. It makes it easy to bring in pre-configured CSS but still gives you the slick tailwind capabilities.

**Pull request merged**